kubernetes-client-api/pom.xml

```diff
@@ -139,6 +139,7 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
+      <version>${jackson.databind.version}</version>
```

Comment:
if the version needs to be setted somewhere, it should be in the parent pom `depedencyManagement` section or bom.